public/js/util/DevToolsUtils.js

```diff
@@ -0,0 +1,13 @@
+function reportException(who, exception) {
+  let msg = who + " threw an exception: ";
+  console.error(msg, exception);
+}
```

Comment:
moved reportException over as I'd like to be able to control how exceptions are reported. This will need to be flagged off in production, which I'll do when i work on configs next.